test/metabase/db/custom_migrations_test.clj

```diff
@@ -1942,3 +1942,42 @@
                                   :uploads_schema_name  nil
                                   :uploads_table_prefix nil}}
                   (m/index-by :id (t2/select :metabase_database)))))))))
+
+(deftest create-sample-content-test
+  (testing "The sample content is created iff *create-sample-content*=true"
+    (doseq [create? [true false]]
+      (testing (str "*create-sample-content* = " create?)
+        (impl/test-migrations "v50.2024-05-27T15:55:22" [migrate!]
+          (let [sample-content-created? #(boolean (not-empty (t2/query "SELECT * FROM report_dashboard where name = 'E-commerce insights'")))]
```

Comment:
Why not define this as a top-level helper function, instead of repeating its definition in each test case?